src/adeft/modeling/classify.py

```diff
@@ -424,7 +429,7 @@ def feature_importances(self):
         output = {}
         tfidf = self.estimator.named_steps['tfidf']
         logit = self.estimator.named_steps['logit']
-        feature_names = tfidf.get_feature_names()
+        feature_names = tfidf.get_feature_names_out()
```

Comment:
This method changed names in scikit-learn 1.0, the old one was deprecated and then removed in 1.2.